### ISSUE TITLE
Test all the different book ideas against the Write Useful Books book

### ISSUE THREAD

**Issue by fmvilas** [fmvilas/shapeup#20]

**Comment by fmvilas** [comment]
/progress 40

Tested the following ideas:
1. **Making sense of your event-driven architectures**: sounds awesome for me to write but I think it may get little interest as it will be mainly focused on people managing big EDAs.
1. **The AsyncAPI & Kafka Handbook**: a practical handbook with lots of tips and tricks to help you set up AsyncAPI with your Kafka EDA. This can become a repeatable pattern by simply replacing Kafka with another technology.
1. **From side-project to industry standard**: a book telling my story building AsyncAPI. Less about technical aspects and more about all the stories that happened and how we built the community. I love this but decided to postpone it as I think we're still far from done here!

So far seems that number 2 is winning but I'll sleep over it.
